tools/pfe-tools/11ty/DocsPage.ts

```diff
@@ -85,6 +85,8 @@ export class DocsPage implements DocsPageRenderer {
     this.templates.addFilter('log', DocsPage.#log);
     this.templates.addFilter('type', DocsPage.#type);
     this.templates.addFilter('innerMD', DocsPage.#innerMD);
+    this.templates.addFilter('mdHeading', (header, length = 2) =>
+      DocsPage.#innerMD(`${Array.from({ length }, () => '#').join('')} ${header}`));
```

Comment:
Super minor suggestion, not important.

```suggestion
    this.templates.addFilter('mdHeading', (header, level = 2) =>
      DocsPage.#innerMD(`${Array.from({ level }, () => '#').join('')} ${header}`));
```

Reply:
length is a keyword here. Array.from can construct an array from an object with a length property

Reply:
👀 🤯 very cool also TIL.